README.md

```diff
@@ -159,6 +159,30 @@ installation and massively reducing the install time on CI.
 When using the [Nix](https://nixos.org/nix) package manager, the `nixpkgs.cargo-tarpaulin` package can be used.
 This ensures that tarpaulin will be built with the same rust version as the rest of your packages.
 
+### Environment Vars
+
+Tarpaulin sets the below environment vars under test:
```

Comment:
change "vars" to "variables"

Reply:
Additionally, I think it's worthwhile to add a bit more run in, just to mention that tarpaulin tries to set up an equivalent environment to when you run your tests via cargo test i.e. 

> When tarpaulin runs your tests it strives to run them in the same environment as if they were ran via `cargo test`. In order to achieve this it sets the following environment variables when executing the test binaries:

This is only a rough wording I haven't given it too much thought. If you think you have a nicer phrasing I'm open to that as well :smile:

Reply:
Thanks incorporated

Reply:
I tend to use the env everywhere for some reason, good catch :)